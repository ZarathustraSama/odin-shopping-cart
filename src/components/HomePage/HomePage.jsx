import NavigationBar from "../NavigationBar/NavigationBar";

export default function HomePage() {
  <>
    <NavigationBar></NavigationBar>
  </>
}