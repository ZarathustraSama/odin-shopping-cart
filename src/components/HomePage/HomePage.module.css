.h1 {
  text-align: center;
}

.p {
  font-size: x-large;
  text-align: center;
}