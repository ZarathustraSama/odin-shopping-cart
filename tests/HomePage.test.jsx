import { describe, it, expect } from 'vitest';
import { render } from "@testing-library/react";
import HomePage from '../src/components/HomePage/HomePage';
import { BrowserRouter } from 'react-router-dom';

describe("HomePage", () => {
  it("should render a NavBar component, a header and a paragraph", () => {
    const { container } = render(<BrowserRouter><HomePage /></BrowserRouter>);
    expect(container).toMatchSnapshot();
  })
})