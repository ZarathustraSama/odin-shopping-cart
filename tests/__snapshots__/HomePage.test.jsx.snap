// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomePage > should render a NavBar component, a header and a paragraph 1`] = `
<div>
  <nav>
    <a
      data-discover="true"
      href="/"
    >
      Home
    </a>
    <a
      data-discover="true"
      href="/shop"
    >
      Shop
    </a>
    <span
      role="textbox"
    >
      Your cart currently has 
      0
       items. 
    </span>
  </nav>
  <h1>
    Welcome to this incredible service that doesn't expire!
  </h1>
  <p>
    This is just a fac-simile for an e-commerce site. Feel Free to navigate the pages via the links above and take a look!
  </p>
</div>
`;
